src/main/java/org/spongepowered/common/effect/particle/SpongeParticleHelper.java

```diff
@@ -139,16 +151,22 @@ private static int getDirectionData(Direction direction) {
                 if (effects.isEmpty()) {
                     return Collections.emptyList();
                 }
-                FIREWORK_ROCKET_DUMMY.setPosition(position.getX(), position.getY(), position.getZ());
+                final SPacketSpawnObject packetSpawnObject = new SPacketSpawnObject();
+                packetSpawnObject.entityId = FIREWORK_ROCKET_ID;
+                packetSpawnObject.uniqueId = FIREWORK_ROCKET_UNIQUE_ID;
+                packetSpawnObject.x = position.getX();
+                packetSpawnObject.y = position.getY();
+                packetSpawnObject.z = position.getZ();
+                packetSpawnObject.type = 76;
```

Comment:
Where does this number come from? A comment on this line would be nice for anyone looking at this in the future.